.github/workflows/ci.yaml

```diff
@@ -0,0 +1,35 @@
+name: Go test
+
+on:
+  pull_request:
+
+jobs:
+  run-test:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go: [ '1.19' ]
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: ${{ matrix.go }}
+          cache: true
+        
+      - name: Install goimports
+        run: go get golang.org/x/tools/cmd/goimports
```

Comment:
If I'm not mistaken, go 1.18 remove the ability to use `go get` for tools? I think you might need to use `go install` here.

Reply:
Go 1.18 did indeed remove the ability to use `go get` for this: https://go.dev/doc/go-get-install-deprecation
So will need to switch to `go install` most likely.